docs/modules/qbi-helper/examples.md

```diff
@@ -0,0 +1,124 @@
+<%(TOC)%>
+
+# Examples
+
+## Minimap
+
+Consider a simple game that displays a minimap to players, with three components:
+
+* `PlayerControls`, authoritative on client-worker
+* `PlayerInfo`, authoritative on server-worker
+* `MinimapRepresentation`, authoritative on server-worker
+
+<%(#Expandable title="See schema")%>
```

Comment:
Is the schema actually relevant to the example?

Reply:
Similar comment for the other example

Reply:
It is not relevant - will remove